agent/engine/docker_container_engine.go

```diff
@@ -271,14 +274,19 @@ func (dg *dockerGoClient) pullImage(image string, authData *api.RegistryAuthenti
 
 	// Special case; this image is not one that should be pulled, but rather
 	// should be created locally if necessary
-	if image == emptyvolume.Image+":"+emptyvolume.Tag {
+	if image == fmt.Sprintf(imageNameFormat, emptyvolume.Image, emptyvolume.Tag) {
 		scratchErr := dg.createScratchImageIfNotExists()
 		if scratchErr != nil {
 			return CreateEmptyVolumeError{scratchErr}
 		}
 		return nil
 	}
 
+	// Skip pull pause container image, it's already loaded on agent restart
```

Comment:
Can you remove this logic for now?  I think I'd rather have this occur at a different layer in the codebase (probably in DockerTaskEngine rather than here, so we can trigger on the container type rather than the image name) and it's orthogonal to the rest of the PR.  I was planning on looking at this on Monday.

Reply:
removed